lib/resources/styles.css

```diff
@@ -562,6 +562,17 @@ button {
     margin-right: 24px;
   }
 
+  #overlay-under-drawer.active {
+    opacity: 0.7;
+    width: 100%;
```

Comment:
I would. I don't really trust percentages w/ the dom. I think the fixed positioning will be more reliable.